index.html

```diff
@@ -1,8 +1,100 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang = "en">
   <head>
-    <meta charset="UTF-8">
+    <meta charset = "UTF-8">
+    <meta name = "viewport" content = "width = device-width, initial-scale = 1">
     <title>Fork Me! FCC: Test Suite Template</title>
-    <link rel="stylesheet" href="css/style.css">
+    <link rel = "stylesheet" href = "css/style.css">
   </head>
+
+  <body>
+    <h1 id = 'title'>freeCodeCamp Survey Form</h1>
+    <p>Thank you for taking the time to help us improve the platform</p>
+
+    <div>
+      <form id = 'survey-form'>
+        <div>
+          <label for = "name" style = "color: red">Name</label><br>
```

Comment:
- [] If you have realized, you have linters errors because of using inline styling. Using inline styling is not a good practice. Kindly clean all the inline styling in your code, and move them to the style.css file.